src/compas_slicer/print_organization/base_print_organizer.py

```diff
@@ -131,45 +151,18 @@ def get_printpoint_neighboring_items(self, layer_key, path_key, i):
 
     def printout_info(self):
         """Prints out information from the PrintOrganizer"""
-
-        layers = 0
-        paths = 0
-        total_number_of_pts = 0
-        total_time = 0
-        total_length = 0
-        flat_dict = []
-
-        for layer_key in self.printpoints_dict:
-            layers += 1
-            for path_key in self.printpoints_dict[layer_key]:
-                paths += 1
-                for printpoint in self.printpoints_dict[layer_key][path_key]:
-                    flat_dict.append(printpoint)
-                    total_number_of_pts += 1
-
-        for i in range(len(flat_dict)):
-            curr = flat_dict[i]
-            prev = flat_dict[i - 1]
-
-            if i > 0:
-                # calculate length of toolpath
-                length = distance_point_point(prev.pt, curr.pt)
-                total_length += length
-                # get speed for every section and calculate time
-                if curr.velocity:
-                    speed = curr.velocity
-                    time = length / speed
-                    total_time += time
-
-        min, sec = divmod(total_time, 60)
-        hour, min = divmod(min, 60)
+        ppts_attributes = {}
+        for key in self.printpoints_dict['layer_0']['path_0'][0].attributes:
+            ppts_attributes[key] = str(type(self.printpoints_dict['layer_0']['path_0'][0].attributes[key]))
 
         print("\n---- PrintOrganizer Info ----")
-        print("Number of layers: %d" % layers)
-        print("Number of paths: %d" % paths)
-        print("Number of PrintPoints: %d" % total_number_of_pts)
-        print("Toolpath length: %d mm" % total_length)
-        print("Printing time: %d hours, %02d min, %02d sec" % (hour, min, sec))
+        print("Number of layers: %d" % self.number_of_layers)
+        print("Number of paths: %d" % self.number_of_paths)
+        print("Number of PrintPoints: %d" % self.number_of_printpoints)
+        print("PrintPoints attributes: ")
+        for key in ppts_attributes:
+            print('     % s : % s' % (str(key), ppts_attributes[key]))
+        print("Toolpath length: %d mm" % self.total_length_of_paths)
```

Comment:
Hello! Yes, I cleaned up the printout_info. The time was returning always 0.0 since it needs the print speed to do the caclulation. But the print speed is fabrication-related information, that would never be part of the slicer, so we won't really be able to calculate actual printing speed, at least not in that place. However, the length of the paths gives that information no? If someone has sliced an object and they know the speed that they are printing with, then finding the time should be trivial. The total speed could go to the print organizer if we want to keep it in, but then we would also have to calculate traveling time to have a good estimation.

Reply:
I could add the placeholder to the print_organiser, I think that makes more sense. Only then we would have to re-caclulate the paths length. What do you think?

Reply:
Aha but the changes you made to the printout_info are in the print organizer right? And in there usually the PrintPoints have a velocity assigned to them, which means you can calculate the printing time. That's why there used to be these lines:

```
            if i > 0:
                # calculate length of toolpath
                length = distance_point_point(prev.pt, curr.pt)
                total_length += length
                # get speed for every section and calculate time
                if curr.velocity:
                    speed = curr.velocity
                    time = length / speed
                    total_time += time

        min, sec = divmod(total_time, 60)
        hour, min = divmod(min, 60)
```

The total toolpath length is also nice to have, but in case you have different printing velocities for different part of the print (for example faster for travel or slower for the first X layers) it is not trivial anymore to calculate the printing time.

Reply:
Ah sorry, I was confused! Yes, u are right. Ok I'll update it

Reply:
Cool :) Otherwise it looks good to merge, thanks for cleaning up also!